person.rb

```diff
@@ -17,6 +18,8 @@ def of_age?
     @age >= 18
   end
 
+  public
```

Comment:
[OPTIONAL] To save you some lines of code, you do not need to indicate a public method as a method is public by default unless it has been indicated otherwise. Hence, you can take away this line and still have the same result😉.

Reply:
@Ibizugbe thank you for the feedback. Suggested change has been implemented. 😉